src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { Component, MouseEvent } from 'react';
+
+export type tabType = 'basic';
+
+interface IProps {
+  headings: string[];
```

Comment:
right now the default active tab is the first heading string in the array.  
If the user would like to set the second heading to be the default active one, how could the user implement it?

Reply:
they can edit in the state?  Like following:
activeHeading:
        this.props.headings && this.props.headings.length > 0
          ? this.props.headings[0]  ** change this to [1]**
          : '', 

Or is there other way to allow user modify it?